### ISSUE TITLE
google Colab getting "no module named 'dateparser'" 

### ISSUE THREAD

**Issue by dominicdev** [obsei/obsei#229]
**Describe the bug**

installing 
!pip install obsei

and run twitter scrape

`[/usr/local/lib/python3.7/dist-packages/obsei/misc/utils.py](https://localhost:8080/#) in <module>()
      3 import time
      4 
----> 5 import dateparser
      6 from datetime import datetime, timezone
      7 from importlib import import_module

ModuleNotFoundError: No module named 'dateparser'`

from obsei.source.twitter_source import TwitterCredentials, TwitterSource, TwitterSourceConfig

**Comment by dominicdev** [comment]
okay its seem to be okay now after 
!pip install dateparser

but im not getting any in the source.. any idea?

from obsei.source.twitter_source import TwitterCredentials, TwitterSource, TwitterSourceConfig

# initialize twitter source config
source_config = TwitterSourceConfig(
   keywords=["#LeeSungKyung"], # Keywords, @user or #hashtags
   lookup_period="10d", # Lookup period from current time, format: `<number><d|h|m>` (day|hour|minute)
   cred_info=TwitterCredentials(
       # Enter your twitter consumer key and secret. Get it from https://developer.twitter.com/en/apply-for-access
       consumer_key="xxxxxxxxxxxxxxxxxxxxxxx",
       consumer_secret="xxxxxxxxxxxxxxxxxxxxxx",
       bearer_token='xxxxxxxxxxxxxxxxxxxxxxx',
   )
)
source = TwitterSource()

**Comment by lalitpagaria** [comment]
@dominicdev Thank you for reporting this. Hope you are doing this after initializing the source?
```
# This will fetch information from configured source ie twitter, app store etc
source_response_list = source.lookup(source_config)
```
`source_response_list` will give you all collected tweet data.

**Comment by dominicdev** [comment]
> @dominicdev Thank you for reporting this. Hope you are doing this after initializing the source?
> 
> ```
> # This will fetch information from configured source ie twitter, app store etc
> source_response_list = source.lookup(source_config)
> ```
> 
> `source_response_list` will give you all collected tweet data.

yes tried it, but getting 
"title": "Unauthorized",
  "type": "about:blank",
  "status": 401,
  "detail": "Unauthorized"

any idea? I already setup my  consumer_key,consumer_secre, bearer_token, what do I missed?

**Comment by lalitpagaria** [comment]
Obsei only supports V2 APIs. Hopefully, you are generating a v2 API client. (Refer image `V2 Access` should be there on client)
![image](https://user-images.githubusercontent.com/19303690/157806207-910fe184-1082-4fac-9116-3d2ded54dc07.png)

Also, try passing only `consumer_key,consumer_secret`, let Obsei fetch `bearer_token`.

**Comment by dominicdev** [comment]
okay will try, thanks

**Comment by dominicdev** [comment]
> Obsei only supports V2 APIs. Hopefully, you are generating a v2 API client. (Refer image `V2 Access` should be there on client) ![image](https://user-images.githubusercontent.com/19303690/157806207-910fe184-1082-4fac-9116-3d2ded54dc07.png)
> 
> Also, try passing only `consumer_key,consumer_secret`, let Obsei fetch `bearer_token`.

how to i know if was V2 im not getting the image you have only got this 
![image](https://user-images.githubusercontent.com/3959917/157809389-76dec6e5-f52a-4cf2-a9db-4416a4e36dda.png)

**Comment by dominicdev** [comment]
![image](https://user-images.githubusercontent.com/3959917/157810685-70b295cc-4201-4065-a55a-ae6a9f2485f8.png)

**Comment by lalitpagaria** [comment]
Please check inside settings. There might be an option to enable V2 access.

**Comment by dominicdev** [comment]
> Please check inside settings. There might be an option to enable V2 access.

yes it was anabled now or need to wait a few minutes to take effect?
![image](https://user-images.githubusercontent.com/3959917/157812189-c3fdb62b-bacd-455a-9759-3125fb57bed3.png)

**Comment by dominicdev** [comment]
its okay now just lower the look up fix the issue

**Comment by lalitpagaria** [comment]
Fixed in #257
